### PR TITLE
PEPPER-1423 & PEPPER-1425 pecgs home page update

### PR DESCRIPTION
**Pull request by ssettipalli**
PEPPER-1423 
lms and osteo home page updates
No specify styling was mentioned and went ahead with my own stying since new text was supposed to be in separate block and not in any existing ones. 
Open to suggestions if styling need to be changed. 

PEPPER-1425
pe-cgs home page text changes

### CODE REVIEW

**Comment by ssettipalli** [comment]
Screenshot 2024-07-12 at 2 03 46 PM
Screenshot 2024-07-12 at 2 05 46 PM

File: ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json [hunk 1]
```diff
@@ -839,6 +839,7 @@
   "home": {
     "paragraph1Header": "Have a direct impact on the future of leiomyosarcoma.",
     "paragraph1Text": "If you have ever been diagnosed with leiomyosarcoma, join a nationwide movement of patients to contribute your samples, your medical records, and your voice to leiomyosarcoma research. Together, we can speed the development of future therapies.",
+    "fundingText": "Count Me In’s Osteosarcoma and Leiomyosarcoma Projects are made possible by federal funding through the Participant Engagement and Cancer Genome Sequencing (PE-CGS) grant as part of the Cancer Moonshot Initiative. Learn more about PE-CGS <a class=\"hereLink\" href=\"https://pe-cgs.org/\" target=\"_blank\"><u>here.</u></a>",
```
Comment (ssettipalli) [inline ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json:842]: wondered if both pec-cgs study names should be there in the text OR only lms

File: ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json [hunk 2]
```diff
@@ -198,6 +198,7 @@
               "Alt": "PE-CGS logo"
             }
         },
+        "fundingText": "Count Me In’s Osteosarcoma and Leiomyosarcoma Projects are made possible by federal funding through the Participant Engagement and Cancer Genome Sequencing (PE-CGS) grant as part of the Cancer Moonshot Initiative. Learn more about PE-CGS <a class=\"hereLink\" href=\"https://pe-cgs.org/\" target=\"_blank\"><u>here.</u></a>",
```
Comment (ssettipalli) [inline ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json:201]: wondered if both pec-cgs study names should be there in the text OR only osteo

**Comment by ssettipalli** [comment]
Screenshot 2024-07-16 at 8 17 49 AM
Screenshot 2024-07-16 at 8 18 38 AM
Screenshot 2024-07-16 at 8 20 13 AM

File: ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json (hunk 1 above)
Comment (zyme) [reply ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json:842]: I think it's both, but check with Elana.

**Review by zyme — APPROVED** [review]

File: ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json (hunk 1 above)
Comment (ssettipalli) [reply ddp-workspace/projects/ddp-lms/src/assets/i18n/en.json:842]: Since spanish (OS only has Spanish) text also had both, I believe its both for sure

**Pull request merged**